cpp/src/io/parquet/compact_protocol_reader.cpp

```diff
@@ -140,6 +140,7 @@ struct parquet_field_bool_list : public parquet_field_list<bool, FieldType::BOOL
     auto const read_value = [&val = v](uint32_t i, CompactProtocolReader* cpr) {
       auto const current_byte = cpr->getb();
       assert_bool_field_type(current_byte);
+      CUDF_EXPECTS(i >= 0 and i < val.size());
```

Comment:
Here `i` is already unsigned.
```suggestion
      CUDF_EXPECTS(i < val.size());
```

Reply:
Fixed in 86b594a.